src/test/java/uk/ac/ebi/embl/api/validation/ValidationMessageTest.java

```diff
@@ -122,4 +123,13 @@ public void testWriteTextMessageChangedFormatter() throws IOException {
       ValidationMessage.setDefaultMessageFormatter(mf);
     }
   }
+
+  @Test
+  public void testValidationScopeWithGroup() {
+    assertTrue(ValidationScope.NCBI.isInGroup(ValidationScope.Group.NCBI));
```

Comment:
Please move the validation scope and group tests into a new class:
```
ValidationScopeTest
```
and test that all ValidationScope enums against isInGroup:

```
@Test
public void testValidationScopeIsInGroup() { 
assertTrue(ValidationScope.NCBI.isInGroup(ValidationScope.Group.PUTFF));
assertFalse(ValidationScope.NCBI.isInGroup(ValidationScope.Group.PIPELINE));
etc.
...
} 
```